README.md

```diff
@@ -17,8 +18,10 @@ eksporter miljøvariabel NPM_TOKEN, f eks ved å legge til
 ## Kjør lokalt
 1. `yarn install`
 2. `yarn start:dev`
+3. Kjør opp familie-ba-soknad-api
 
-For å kjøre med mellomlagring må du ha familie-dokument kjørende.
+### Mellomlagring
+For å kjøre med mellomlagring må du ha familie-dokument kjørende (https://github.com/navikt/familie-dokument).
```

Comment:
Kan evt få med at denne må kjøre "DevLauncher" og ikke "DevLauncherMedMockServer"

Reply:
Noe jeg må gjøre for å slippe 401?
![image](https://user-images.githubusercontent.com/17828446/191035420-3a413961-d1b8-47dc-97b7-41c1f8b48cab.png)

Reply:
Hehe kan skrive at man må kjøre "DevLauncherForSøknad" og ikke "DevLauncher" eller "DevLauncherMedMockServer" 🤓

Reply:
Det står egentlig i read-me'en til familie-dokument allerede 🙈 Synes du vi trenger å ha det begge steder?

![Screenshot 2022-09-20 at 15 23 00](https://user-images.githubusercontent.com/8656966/191269244-df6dd6cd-57d5-460b-9bb1-dc41ff24dc37.png)